docs/RELEASE_NOTES.md

```diff
@@ -145,6 +156,7 @@ the library. `1.0.0` is here for you to use with all new features, enhancements
     - **Commons** - Bumped `org.junit.jupiter:junit-jupiter-engine` to 5.3.2.
     - **Commons** - Bumped `org.junit.jupiter:junit-jupiter-params` to 5.3.2.
     - **Commons** - Bumped `org.ajoberstar.git-publish` to 2.0.0.
+    - **Commons** - Bumped `com.adarshr.test-logger` to 1.6.0.
```

Comment:
why is this added again? I see it already here: https://github.com/commercetools/commercetools-sync-java/pull/399/files#diff-aa75d2e7837774f5cb565717b6744a35R154

Reply:
aha, nice catch, a merge conflict issue 🙁
6d23157